forms-flow-theme/scss/_forms.scss

```diff
@@ -189,3 +189,56 @@ select option:hover {
   line-height: 120%;
 }
 
+/* CustomTooltip */
+.custom-tooltip .tooltip-inner {
+  background-color: var(--ff-base-100);
+  color: var(--ff-gray-800);
+  font-size: var(--font-size-xs);
+  font-weight: var(--font-weight-sm);
+  line-height: var(--text-line-height);
+  white-space: nowrap;
+  padding: 0.5rem;
+  border-radius: 0.5rem;
```

Comment:
please use spacing and border variables on all places

Reply:
added